packages/@sanity/base/src/components/IntentButton.tsx

```diff
@@ -11,5 +11,5 @@ export function IntentButton(
     params: Record<string, string>
   }
 ) {
-  return <Button {...props} as={IntentLink} />
+  return props.disabled ? <Button {...props} as="a" /> : <Button {...props} as={IntentLink} />
```

Comment:
A link without `href` doesn't have an implicit `link` role, so I suggest we add these attributes:

```suggestion
  return props.disabled ? <Button {...props} as="a" role="link" aria-disabled="true" /> : <Button {...props} as={IntentLink} />
```

[Source](https://www.scottohara.me/blog/2021/05/28/disabled-links.html)

Reply:
Big brain! 🧠